bundle/edu.gemini.pot/doc/GhostEstimates.md

```diff
@@ -0,0 +1,610 @@
+# GHOST Estimations for High-Level Software
+
+GHOST brings an important new feature to the science program model, the ability to observe two targets simultaneously.  All previous instruments are limited to a single science target per observation.  The purpose of this document is to sketch out ideas for how to implement this feature along with the rest GHOST and then make time estimates for doing the work.
+
+The GHOST concept of operations document, section 4.2 “Instrument Operating Modes”, describes a sophisticated set of modes that use the two IFUs with built in guiding capability and dedicated sky fibers in distinct ways. This prescribes a tight level of integration between target definition and instrument features that is unprecedented for Gemini instruments. A few additional facts about dual target support may be relevant to implementation ideas:
+
+* The two target mode applies only to GHOST, and only to GHOST in a particular mode of operations: Dual Target Standard Resolution.  There are other ways to use GHOST that occupy a single target, just as for all other instruments.
+
+* Because there are two IFUs, in beam-switching and high-resolution modes one can be observing a science target and the other taking sky background at a specific coordinate.
+
+* The two science targets are limited to sidereal targets.  Dual non-sidereal and ToO targets are not supported.
+
+* The concept of a base position to be used for slewing and guiding calculations remains.  The base position in dual target mode is the coordinate that is equidistant from the two targets.  In other words, the point in the middle of a line connecting the two targets.
+
+* There is a minimum (84 arcseconds) and maximum (7 arcminutes) separation of the two targets.
+
+* Future instruments after GHOST are also likely to support multiple targets, each with their own rules for calculating the base position and placement.
+
```

Comment:
It might be useful to add another point here that the IFUs are distinct and the assignment of targets to one or the other is not arbitrary. This may be obvious in retrospect but wasn't initially and it ended up driving the design.